mtp_noms_ops/templates/security/accept_or_reject_check.html

```diff
@@ -182,9 +182,23 @@ <h3 class="heading-medium">{% trans 'Credit decisions matching debit card or pri
 
         <div class="form-group">
           <h2 class="heading-medium">{% trans 'Accept or reject this credit' %}</h2>
-          <h3 class="heading-medium">{% trans 'To accept this credit:' %}</h2>
+          <h3 class="heading-medium">{% trans 'To accept this credit:' %}</h3>
           <div>
             {% include 'mtp_common/forms/textarea.html' with field=form.accept_further_details input_classes='form-control-3-4' %}
+            <div class="multiple-choice form-group" data-target="auto_accept_reason">
+              <input id="example-auto_accept_reason" class="js-ToggleTextFieldOnChange" type="checkbox" name="checkbox-rejection-group" value="No">
+              <label for="example-auto_accept_reason">
+                {% blocktrans trimmed with sender_name=check.credit.sender_name prisoner_number=check.credit.prisoner_number %}
+                  Automatically accept future credits from {{ sender_name }} to {{ prisoner_number }}
+                {% endblocktrans %}
+                <br/>
+                {% trans 'They will be flagged in Decision history.' %}
+              </label>
+            </div>
+            <div class="panel panel-border-narrow js-hidden" id="auto_accept_reason">
+              <label class="form-label" for="rejection-auto_accept_reason">{% trans 'Give reason for automatically accepting' %}</label>
+              <input class="form-control" name="auto_accept_reason" type="text" id="rejection-auto_accept_reason" disabled>
```

Comment:
Maybe it's a stupid question (sorry!) but any reason for this input's ID being `rejection-auto_accept_reason`, does the ID starts with `rejection-` because of some styling or Form naming convention reason maybe?

Reply:
it doesn't really matter what the id is. it's not used for styling or scripting, just to link the label to the input